src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+    public Ball(String color, int number) {
```

Comment:
return empty lines) why did you delete them?

Reply:
trying to hard)))))